### ISSUE TITLE
Backport for TBC

### ISSUE THREAD

**Issue by Macumbafeh** [rgd87/NugEnergy#7]
Hello,
Is it possible to backport it for TBC please?

EnergyWatch is so ugly and it seems you did a great job with this Addon.

Feel free to join our Addon Discord TBC 2.4.3 if you wanna talk with our team : 
https://discord.gg/5qVu56M

Thanks!

**Comment by d87** [comment]
No, but I can give you an actual 2.4.3 version of this addon...
[NugEnergy_2.4.zip](https://github.com/rgd87/NugEnergy/files/4673387/NugEnergy_2.4.zip)

It's probably very ugly too lol. but maybe customizable enough

**Comment by Macumbafeh** [comment]
Thanks. I'll give it a try!

**Comment by Macumbafeh** [comment]
If your other addons have 2.4.3 version too i'm down :).
I have already NugComboBar.

**Comment by d87** [comment]
No, first NugRunning release was for 3.0

**Comment by Macumbafeh** [comment]
Very Nice addon NugEnergy thanks so much ! even if for you is ugly it's better than EnergyWatch xD
